app/src/utils/get-asset-url.ts

```diff
@@ -0,0 +1,6 @@
+import { addTokenToURL } from '@/api';
+import { getRootPath } from '@/utils/get-root-path';
+
+export function getAssetUrl(filename: string, isDownload?: boolean): string {
+	return addTokenToURL(getRootPath() + `assets/${filename}` + (isDownload ? '?download' : ''));
```

Comment:
Lets use a URL object to make these modifications, and add some unit tests 🙂

Reply:
unit test writing is very fun :D